Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -902,7 +902,9 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
      * - Note: This feature is experimental and may change or be removed in the
      * future.
      */
-    #warning("Deprecate this property if #3114 will not be done")
+
+    /// TODO: Deprecate this property if #3114 will not be done
```

Comment:
Switched from `#warning` to `TODO` to get the test configuration projects building because they are configured to treat warnings as errors.